app/frontend/actions.js

```diff
@@ -423,18 +423,23 @@ module.exports = ({setState, getState}) => {
     }
   }
 
-  const resetSendForm = (state) => {
+  const resetSendFormState = () => {
```

Comment:
Even though the value of `state` will not be read, it makes sense to keep `state` as argument to signify that this function mutates it and to keep the style consistent.